packages/react/src/PageLayout/PageLayout.features.stories.tsx

```diff
@@ -162,6 +162,7 @@ export const NestedScrollContainer: StoryFn = args => (
           <Placeholder label="Header" height={64} />
         </PageLayout.Header>
         <PageLayout.Content padding="normal" width="large">
+          {/* eslint-disable-next-line github/a11y-role-supports-aria-props */}
```

Comment:
@TylerJDev can we fix these violations??

Reply:
Looks like it was [turned off in eslint-plugin-github](https://github.com/github/eslint-plugin-github/blob/c3857ba01e1d6b3d14269b6625eb604360e5cc8c/lib/configs/react.js#L11), so I'd say yes!